src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props

```diff
@@ -35,6 +35,35 @@
 
   <!-- For CPS/VS support. Importing in .props allows any subsequent targets to redefine this if needed -->
   <Target Name="CompileDesignTime" />
+    
+	<Target Name="_PackageDatabaseResults" Condition="'$(IsDatabaseProject)'=='True'" BeforeTargets="_GetPackageFiles">
```

Comment:
What does this section do exactly?

Reply:
This control the build dependency order.  When I had multiple projects without this feature they were sometime building out of order and not collecting all the child dependencies when the build was running multithreaded.

Reply:
Hate to be a Pain, but any idea when this will be ready to merge??  Does it work?  Currently I need to pack these projects differently than standard projects until this feature is completed...

Reply:
It worked when I sent the pr a few months back.  I have a work around for my own projects so if you guys don't want to accept the change and own it then no worries from me.